tests/testArgumentParser.py

```diff
@@ -196,12 +210,28 @@ def testShow(self):
             config = self.config,
             args = [DataPath, "--show", "config=X"],
         )
+
+        # Test show with glob for single line doc string
         with capture() as out:
             self.ap.parse_args(self.config, [DataPath, "--show", "config=*strItem*", "run"])
         stdout = out[0]
-        answer = stdout.rstrip().split('\n')
-        self.assertEqual(len(answer), 1)         # only 1 config item matches
-        self.assertTrue("strDefault" in answer[0])
+        stdoutList = stdout.rstrip().split("\n")
+        self.assertGreater(len(stdoutList), 2) # at least 2 docstring lines (1st line is always a \n)
+                                               # and 1 config parameter
+        answer = [ans for ans in stdoutList if not ans.startswith("#")] # get rid of comment lines
+        self.assertEqual(len(answer), 2)       # only 1 config item matches (+ 2 entries per doc string)
+        self.assertIn("strDefault", answer[1])
+
+        # Test show with glob for multi-line doc string
+        with capture() as out:
+            self.ap.parse_args(self.config, [DataPath, "--show", "config=*choiceItem", "run"])
+        stdout = out[0]
+        stdoutList = stdout.rstrip().split("\n")
+        self.assertGreater(len(stdoutList), 2) # at least 2 docstring lines (1st line is always a \n)
+                                               # and 1 config parameter
+        answer = [ans for ans in stdoutList if not ans.startswith("#")] # get rid of comment lines
+        self.assertEqual(len(answer), 2)       # only 1 newline plus 1 config item matches
+        self.assertIn("ONE", answer[1])
```

Comment:
You could refactor the above to reduce quite a lot of duplication (there's a repeating `with capture; assertion; list comprehension; assertion; assertion` pattern, which smells a bit dubious).